### PR TITLE
Fix for JavaScript coding error when alternativeAttribute is set

### PR DESCRIPTION
**Pull request by brian-learningpool**

### CODE REVIEW

File: frontend/src/core/scaffold/views/scaffoldAssetView.js [hunk 1]
```diff
@@ -284,6 +284,7 @@ define(function(require) {
 
             // Check if alternative attribute should be used
             if (alternativeAttribute) {
+                attributesToSave = [];
```
Comment (dennis-learningpool) [inline frontend/src/core/scaffold/views/scaffoldAssetView.js]: should `attributesToSave` only be initialized if it doesn't already exist? Can it exist at this point?

Comment (brian-learningpool) [inline frontend/src/core/scaffold/views/scaffoldAssetView.js]: It's declared at the top of the function, but given a value of `null` in certain cases.

**Comment by dennis-learningpool** [comment]
+1

**Comment by darylhedley** [comment]
+1

**Pull request merged**